core/generator.js

```diff
@@ -402,32 +397,59 @@ Blockly.Generator.prototype.definitions_;
 Blockly.Generator.prototype.functionNames_;
 
 /**
- * A database of variable names.
+ * A database of variable and procedure names.
  * @type {Blockly.Names}
  * @protected
  */
-Blockly.Generator.prototype.variableDB_;
+Blockly.Generator.prototype.nameDB_;
+
+Object.defineProperty(Blockly.Generator.prototype, 'variableDB_', {
+  /**
+   * Getter.
+   * @deprecated 'variableDB_' was renamed to 'nameDB_' (May 2021).
+   * @this {Blockly.Generator}
+   * @return {Blockly.Names} Name database.
+   */
+  get: function() {
+    console.warn('Deprecated use of "variableDB_"; change to "nameDB_"');
```

Comment:
We have the Blockly.utils.deprecation.warn function, which would be good to use for consistency and so we can more easily track what we have scheduled for deletion

Reply:
Fixed, and changed Blockly.utils.deprecation.warn to also handle properties, not just functions.